crates/bdk/tests/wallet.rs

```diff
@@ -1446,7 +1446,9 @@ fn test_bump_fee_reduce_change() {
         .unwrap();
 
     let mut builder = wallet.build_fee_bump(txid).unwrap();
-    builder.fee_rate(FeeRate::from_sat_per_vb(2.5)).enable_rbf();
+    builder
+        .fee_rate(FeeRate::from_sat_per_vb_unchecked(3))
+        .enable_rbf();
```

Comment:
This changes the logic of the test. We are now testing an integer instead of a float.
Why?

Reply:
True, using kwu here to test 2.5 (which would be 625) would probably be better.